src/wp-includes/.wp-env.override.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
That file is definitely here by accident 😅

Reply:
oh my 🤦‍♂️